### PR TITLE
Basalt Website Changes

### PR DESCRIPTION
**Pull request by decodyng**

### CODE REVIEW

File: about/index.md [hunk 1]
```diff
@@ -6,29 +6,61 @@ title: About MineRL
...
 
 Why Minecraft?
 --------------
 
 Minecraft is a rich environment to do RL on: it is an open-world environment, has sparse rewards, and has many innate
-task hierarchies and subgoals. Furthermore, it encompasses many of the problems that we must solve as we move towards
-more general AI (for example, what is the reward structure of "building a house"?). Besides all this, Minecraft has
-more than 90 million monthly active users, making it a good environment on which to collect a large-scale dataset.
+task hierarchies and subgoals. People can have many different goals within the game: perhaps you want to [defeat the Ender
+Dragon while others stop you](https://www.youtube.com/watch?v=tylNqtyj0gs), or [build a giant floating island
```
Comment (decodyng) [inline about/index.md]: Maybe should be "while others try to stop you," since if they just immediately succeed I imagine that isn't very fun :P

Comment (rohinmshah) [reply about/index.md]: Lol, sounds right, fixed

File: about/index.md [hunk 2]
```diff
@@ -6,29 +6,61 @@ title: About MineRL
...
+
+To spur research on open-world environments with human data, we release **MineRL**: a suite of environments within Minecraft,
+alongside a large-scale dataset of human gameplay within those environments.
+
+Besides the challenges discussed above, these environments also highlight a variety of other research challenges, including
+open-world multi-agent interactions, long-term planning, vision, control, navigation, and explicit and implicit subtask
+hierarchies. We also release a flexible framework to define new Minecraft tasks.
 
 
 Papers
```
Comment (brandonhoughton) [inline about/index.md:66]: We should update to add the new papers here, probably in its own PR however

File: index.md [hunk 3]
```diff
@@ -3,12 +3,11 @@ layout: default
...
-### [tl;dr: Get the MineRL dataset!](/docs)
-### [tl;dr: Play Minecraft to benefit Science!](/play)
+### tl;dr: Build AI in Minecraft in the [Diamond](/diamond) and [BASALT](/basalt) competitions!
+### tl;dr: Use the [MineRL environments](/docs)!
+### tl;dr: Get the [MineRL dataset](/dataset)!
 
-Welcome to MineRL. We want to solve [Minecraft](http://minecraft.net) using state-of-the-art Machine Learning! To do so, we have created one of the largest imitation learning datasets with **over 60 million frames** of recorded human player data. Our dataset includes a set of tasks which highlights many of the hardest problems in modern-day Reinforcement Learning: sparse rewards and hierarchical policies. 
+Welcome to MineRL. We want to build agents that play [Minecraft](http://minecraft.net) using state-of-the-art Machine Learning! To do so, we have created one of the largest imitation learning datasets with **over 60 million frames** of recorded human player data. Our dataset includes a set of tasks which highlights many of the hardest problems with current techniques: environments with lots of hierarchy, tasks where rewards are sparse, and tasks where rewards are hard to define.
```
Comment (brandonhoughton) [inline index.md:10]: We should get better numbers on this once we re-render the dataset

File: basalt/index.md [hunk 4]
```diff
@@ -0,0 +1,205 @@
+---
+layout: default
+title: BASALT Competition 2021
+---
+
+## [Sign-up to participate on <u>[AIcrowd]</u>!](https://www.aicrowd.com/challenges/neurips-2020-minerl-competition)
```
Comment (brandonhoughton) [inline basalt/index.md]: This points to the 2020-minerl-competition?

**Review by brandonhoughton — CHANGES_REQUESTED** [review]

File: basalt/index.md (hunk 4 above)
Comment (rohinmshah) [reply basalt/index.md]: Omg good catch, fixed

**Review by brandonhoughton — APPROVED** [review]

**Pull request merged**